packages/neon/neon/lib/src/blocs/accounts.dart

```diff
@@ -114,10 +128,14 @@ class AccountsBloc extends Bloc implements AccountsBlocEvents, AccountsBlocState
     }());
   }
 
+  /// Sets the active account.
+  ///
+  /// If no [account] is specified it will fallback to the first account in [accounts].
+  /// When no account is available [activeAccount] will be set to null triggering a logout.
   @override
-  void setActiveAccount(final Account? account) {
-    final as = accounts.value;
-    activeAccount.add(account ?? as.firstOrNull);
```

Comment:
I don't understand why this is here in the first place. If a null account is passed then it should probably just set to be null.

Reply:
You're right, Just looking at it here on GH we could probably remove it.

Ideally I'd like a private method `_setActiveAccount(final Account? account)` and a public one `setActiveAccount(final Account account)` this way we can still log users out internally of the class by passing null to the private method but external callers can only switch the account and would need to call `removeAccount` to achieve logouts. Otherwise someone could use `setActiveAccount` for logout wich is just wrong.